hack/aks/Makefile

```diff
@@ -256,6 +248,17 @@ dualstack-overlay-byocni-up: rg-up overlay-net-up ## Brings up an dualstack Over
 		--yes
 	@$(MAKE) set-kubeconf
 
+windows-nodepool-up: ## Add windows node pool
+	$(AZCLI) aks nodepool add -g $(GROUP) -n npwin \
+		--node-count $(NODE_COUNT) \
+		--node-vm-size $(VM_SIZE) \
+		--cluster-name  $(CLUSTER) \
+		--os-type Windows \
+		--os-sku $(WINDOWS_OS_SKU) \
+		--max-pods 250 \
```

Comment:
should we set a var for max pods?

Reply:
Currently we do not test anything against the max pods on a node. This is set because the default was 30 and we needed to be able to do scale tests ect. Until we test against it I think it should stay at the maximum, so it matches what the other cluster commands default to.